cli/src/main/scala/com/lightbend/rp/reactivecli/annotations/Annotations.scala

```diff
@@ -61,6 +63,7 @@ object Annotations {
   def apply(labels: Map[String, String], args: GenerateDeploymentArgs): Annotations = {
     val appVersion = version(labels)
     Annotations(
+      namespace = namespace(labels).orElse(namespace(args)),
```

Comment:
should it be namespace args first, consistent with the others here?

Reply:
Actually, you are right - thanks for catching that!